src/lib/y2ntp_client/widgets/main_widgets.rb

```diff
@@ -0,0 +1,254 @@
+require "yast"
+
+require "cwm/widget"
+require "cwm/table"
+
+require "y2ntp_client/dialog/pool"
+
+Yast.import "Confirm"
+Yast.import "NtpClient"
+Yast.import "Popup"
+
+module Y2NtpClient
+  module Widgets
+    # Widget for netconfig policy
+    class PolicyCombo < CWM::ComboBox
+      def initialize
+        textdomain "ntp-client"
+      end
+
+      def label
+        # TRANSLATORS: label for widget that allows to define if ntp configiration is only
+        # from its source or dynamically extended e.g. via DHCP
+        _("Configuration Source")
+      end
+
+      def help
+        # TODO: not written previously, but really deserve something
+        _("<b>Configuration Source</b> defines if network itself can adds ntp sources.")
+      end
+
+      def opt
+        [:editable]
+      end
+
+      def items
+        items = [
+          # combo box item
+          ["", _("Static")],
+          # combo box item
+          ["auto", _("Dynamic")]
+        ]
+        current_policy = Yast::NtpClient.ntp_policy
+        if !["", "auto"].include?(current_policy)
+          items << [current_policy, current_policy]
+        end
+
+        items
+      end
+
+      def init
+        self.value = Yast::NtpClient.ntp_policy
+      end
+
+      def store
+        return if value == Yast::NtpClient.ntp_policy
+
+        log.info "ntp policy modifed to #{value.inspect}"
+        Yast::NtpClient.modified = true
+        Yast::NtpClient.ntp_policy = value
+      end
+    end
+
+    # Widget to configure how ntp will be started
+    class NtpStart < CWM::RadioButtons
+      def initialize(replace_point)
+        textdomain "ntp-client"
+
+        @replace_point = replace_point
+      end
+
+      def label
+        _("Start NTP Daemon")
+      end
+
+      def opt
+        [:notify]
+      end
+
+      def items
+        [
+          # radio button
+          ["never", _("Only &Manually")],
+          # radio button
+          ["sync", _("&Synchronize without Daemon")],
```

Comment:
something like "Synchronize periodically" sounds better to me.

Reply:
@mvidner what do you think? this is original naming, so we should change it if benefits comes over confusion of existing users.

Reply:
Previously this option was explained in a help text: https://github.com/yast/yast-ntp-client/pull/99/files/d2c1c227b14535ad05eec969c55a89604345fd4f#diff-c9d99695dbc4159f62ce2c189e3c9f08L40 Do we have a help text now?

Reply:
@mvidner not yet, but I can add it.